docs/pages/build-reference/variables.md

```diff
@@ -65,10 +66,6 @@ The following environment variables are exposed to each build job &mdash; they a
 - `EAS_BUILD_USERNAME` - the username of the user initiating the build (it's undefined for bot users)
 - `EAS_BUILD_WORKINGDIR` - the remote directory path with your project
 
-<!-- TODO: uncomment when remote (managed) version are fully implemented -->
-<!-- - `EAS_BUILD_ANDROID_VERSION_CODE` - Android version code -->
-<!-- - `EAS_BUILD_IOS_BUILD_NUMBER` - iOS Build number -->
```

Comment:
Managed/remote version support is already live but we decided not to document those.